SR2022/2022-04-26-competition-wrap-up.md

```diff
@@ -0,0 +1,61 @@
+---
+to: Student Robotics 2022
+subject: That's a wrap!
+---
+
+![Competition Photo](TODO)
```

Comment:
Photo is uploaded to https://photos.google.com/share/AF1QipOrap74_NX4LT6TYC-qI7Fmyf81uaX_Kqa4LEvGk1pRNzyAoUDWqAZ6nJuMHL2_uQ/photo/AF1QipNpILlVHyMKdq7bIwggmcFG9hI9oj30pX07l3IZ?key=UkQ5aGE5SGplYnJZenFiSWVLamtNc3BjT3VjS2dn